govc/library/item/upload.go

```diff
@@ -106,10 +107,9 @@ func (cmd *upload) Run(ctx context.Context, f *flag.FlagSet) error {
 		}
 
 		// Create a library item
-		cmd.item.Name = filename
+		cmd.item.Name = filepath.Base(filename)
```

Comment:
@akutz realized `library.Finder` (library.{ls,info}) doesn't work if filename includes `/`'s.  But I think we want to use the basename here in any case.

Reply:
Hi @dougm,

That's correct, the finder tokenizes on `/`. If you want to fix this you could just change it so that if `len(parts) > 3` then `parts[2] = strings.Join("", parts[2:])`

Reply:
Hi @dougm,

Please see https://github.com/vmware/govmomi/pull/1381

Reply:
Thanks @akutz , just wanted to mention it for now.  We can include the fix with the vcsim/bats tests PR